src/Form/Type/UserGenderListType.php

```diff
@@ -29,4 +31,22 @@
  */
 class UserGenderListType extends BaseStatusType
 {
+    /**
+     * {@inheritdoc}
+     */
+    public function configureOptions(OptionsResolver $resolver): void
+    {
+        // NEXT_MAJOR: Call the parent to set the choices
+        $choices = call_user_func([$this->class, $this->getter]);
+
+        $resolver->setDefaults([
+            'choices' => array_flip($choices),
```

Comment:
In theory you dont need to flip them again

Reply:
Made some adjustments to detect if we need a flip or not.